packages/terra-clinical-header/src/Header.jsx

```diff
@@ -35,27 +53,44 @@ const propTypes = {
    * A Boolean indicating if element is a subheader.
    */
   isSubheader: PropTypes.bool,
+
+  /**
+   * IDs should be used with 'aria-labelledby' to associate headings with the corresponding page area when needing to label regions.
+   *
+   * For an example, read [Accessibility Guide: Labeling Section Content](/components/terra-clinical-header/clinical-header/accessibility-guide#labeling-section-content) for additional information.
+   */
+  id: PropTypes.string,
 };
 
 const defaultProps = {
   title: '',
   startContent: null,
   endContent: null,
   isSubheader: false,
+  text: '',
 };
 
 const Header = ({
-  children, title, startContent, endContent, isSubheader, ...customProps
+  children, title, startContent, endContent, text, level, id, isSubheader, ...customProps
 }) => {
   const theme = useContext(ThemeContext);
+  if (title) {
+    // eslint-disable-next-line no-console
+    console.warn('`title` prop has been renamed to `text`. please update all the refernces of `title` prop to use prop `text`.'); // to be removed on next major version release.
+  }
+  if (!level) {
+    // eslint-disable-next-line no-console
+    console.warn('Default heading level may not appropriate has it would fail to convey context of heading in a site / application where it is used. Heading level should be set explicitly depending on the position of header in site / application to allow screen readers to identify headers consistently.'); // to be removed on next major version release.
```

Comment:
```suggestion
    // TODO: remove on the next major version bump
    console.warn('Default heading level may not appropriate has it would fail to convey context of heading in a site / application where it is used. Heading level should be set explicitly depending on the position of header in site / application to allow screen readers to identify headers consistently.');
```

Reply:
updated [here](https://github.com/cerner/terra-clinical/pull/823/commits/7cb1cc54d09c6b0f308fdeb4e38f55dd39d88f4c#diff-8c5f8d7501709c71d62d2a51f32a0f10dd712d5a0ede7105f9e0979c4ab845ff)